pkg/api/kubegraph/nodes/types.go

```diff
@@ -234,3 +242,28 @@ func (n SecretNode) String() string {
 func (*SecretNode) Kind() string {
 	return SecretNodeKind
 }
+
+func HorizontalPodAutoscalerNodeName(o *extensions.HorizontalPodAutoscaler) osgraph.UniqueName {
+	return osgraph.GetUniqueRuntimeObjectNodeName(HorizontalPodAutoscalerNodeKind, o)
+}
+
+type HorizontalPodAutoscalerNode struct {
+	osgraph.Node
+	*extensions.HorizontalPodAutoscaler
```

Comment:
> Anonymous types cause us pain, let's not make it worse.

I see as outstanding.  I don't see a reason to continue something that causes us pain.